docs/dev/guidelines/client-design.rst

```diff
@@ -0,0 +1,243 @@
+
+    If you need to separate something from the background, try to use the ``bg-light`` class which should work in both themes.
+
+2. **Define your own colors for each theme**
+
+    | If the options above don't suit your use case, you can define your own color variables.
+    | These colors must be theme-aware, so you have to select a good color for both themes and add them to each
+      theme's stylesheet.
+
+    .. TIP::
+        | Artemis uses ``white`` in light mode and ``$neutral-dark`` in dark mode
+          as background for the main content area, cards etc.
+        |
+        | For ``$neutral-dark``, a few lightened default options exist: ``$neutral-dark-l-5;`` to ``$neutral-dark-l-20;`` in steps of 5.
+        | Therefore, if you need to separate something from the background, choose one of ``gray-XXX`` for light mode and a lightened option of ``$neutral-dark`` in dark mode.
+        |
+        | Keep this in mind while you select the colors to use.
+
+    Let's go through it step by step. Let's say, you want to give a box a special background color.
+
+    1. Define a class for it in your component's SCSS file, and use a new, unique variable name as value:
+
+        .. code-block:: scss
+
+            .my-box {
+                background-color: var(--my-special-component-my-box-background-color);
+            }
+
+    2. Add the variable as SCSS variable to both theme variable sheets (``_default-variables.scss`` and ``_dark-variables.scss``):
+
+        .. code-block:: scss
+
+            // My Special Component
+            $my-special-component-my-box-background-color: [...];
+
+    3. Select a value for each theme.
+
+        | **Re-use preset colors where possible!** For example, choose ``gray-700`` in light mode and ``gray-400`` in dark mode.
+        | This is still a good approach as you-re reusing already provided colors.
+
+        .. code-block:: scss
+
+            // My Special Component
+            $my-special-component-my-box-background-color: $gray-600;
+
+        .. IMPORTANT::
+            The two options below are meant as a fallback for special cases.
+            Please justify the use of these options in your PR description.
+
+        If you need a **slightly modified preset color**, use `SCSS functions <https://sass-lang.com/documentation/modules/color>`_ to derive it.
+
+        .. code-block:: scss
+
+            // My Special Component
+            $my-special-component-my-box-background-color: darken($success, 20);
+
+        **If all of this doesn't work, define your own colors.**
+
+        .. code-block:: scss
+
+            // My Special Component
+            $my-special-component-my-box-background-color: #123456;
+
+1. Theme-specific global styles
+===============================
+
+It might happen that you need to modify a global style rule in one of the themes, for example if you're using a external library which styles need to be overridden.
```

Comment:
```suggestion
It might happen that you need to modify a global style rule in one of the themes, for example if you're using an external library which styles need to be overridden.
```